hazelcast/src/main/java/com/hazelcast/sql/SqlService.java

```diff
@@ -89,6 +89,39 @@ default SqlResult execute(@Nonnull String sql, Object... arguments) {
         return execute(statement);
     }
 
+    /**
+     * Convenient method to execute a distributed query with the given
```

Comment:
"Convenient" -> "Convenience"

We should mention that it executed not any query, but only queries with do not return rows in the beginning of the description.